gui-wm/hyprland/hyprland-9999.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="A dynamic tiling Wayland compositor that doesn't sacrifice on its looks"
+HOMEPAGE="https://github.com/hyprwm/Hyprland"
+
+if [[ "${PV}" = *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN^}.git"
+	EGIT_BRANCH="main"
+	inherit git-r3
+	S="${WORKDIR}/${PN}-${PV}"
+else
+	SRC_URI="https://github.com/hyprwm/${PN^}/releases/download/v${PV}/source-v${PV}.tar.gz -> ${P}.gh.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-source"
```

```suggestion
	SRC_URI="https://github.com/hyprwm/${PN^}/releases/download/v${PV}/source-v${PV}.tar.gz -> ${P}.gh.tar.gz"
	S="${WORKDIR}/${PN}-source"
	
	KEYWORDS="~amd64"
```

usually SRC_URI is followed by S for clarity. Then its better to have KEYWORDS with more space for itself.